src/main/kotlin/no/nav/familie/ef/søknad/service/OppslagServiceServiceImpl.kt

```diff
@@ -13,36 +17,59 @@ import java.time.Period
 
 @Service
 internal class OppslagServiceServiceImpl(private val client: TpsInnsynServiceClient,
+                                         private val featureToggleService: FeatureToggleService,
                                          private val pdlClient: PdlClient,
+                                         private val pdlStsClient: PdlStsClient,
                                          private val regelverkConfig: RegelverkConfig,
                                          private val søkerinfoMapper: SøkerinfoMapper) : OppslagService {
 
     private val secureLogger = LoggerFactory.getLogger("secureLogger")
+    private val logger = LoggerFactory.getLogger(this::class.java)
 
     override fun hentSøkerinfo(): Søkerinfo {
+
+        try {
+            val hentSøkerinfoV2 = hentSøkerinfoV2()
+            if (featureToggleService.isEnabled("familie.ef.soknad.bruk-pdl")) {
+                return hentSøkerinfoV2
+            }
+        } catch (e: Exception) {
+            secureLogger.info("Exception - hent søker fra pdl", e)
+            logger.warn("Exception - hent søker fra pdl (se securelogs for detaljer)")
+        }
+
         val personinfoDto = client.hentPersoninfo()
         val barn = client.hentBarn()
-        secureLogger.info("Personinfo: ${personinfoDto.ident}, Barn alder/samme adresse : ${
-            barn.map { Pair(it.alder, it.harSammeAdresse) }
-        },  ")
         val aktuelleBarn = barn.filter { erIAktuellAlder(it.fødselsdato) }
         return settNavnFraPdlPåSøkerinfo(søkerinfoMapper.mapTilSøkerinfo(personinfoDto, aktuelleBarn))
     }
 
     override fun hentSøkerinfoV2(): Søkerinfo {
-        val søkerinfo = hentSøkerinfo()
-        return settNavnFraPdlPåSøkerinfo(søkerinfo)
+        val pdlSøker = pdlClient.hentSøker(EksternBrukerUtils.hentFnrFraToken())
+        val barnIdentifikatorer = pdlSøker.familierelasjoner
+                .filter { it.relatertPersonsRolle == Familierelasjonsrolle.BARN }
+                .map { it.relatertPersonsIdent }
+        val pdlBarn = pdlStsClient.hentBarn(barnIdentifikatorer)
+        // TODO trenger vi sjekk/filtrering av foreldreansvar
+        val aktuelleBarn = pdlBarn
+                .filter { erIAktuellAlder(it.value.fødsel.first().fødselsdato) }
+                .filter { erILive(it.value) }
```

Comment:
Antar at dere har sjekket dette, men kan man søke tilbake i tid for et barn som ikke lever?

Reply:
Nope, det er ikke noe vi støtter siden tidligere heller.
Burde vi egentlige returnere alle barnen døde/uaktuelle slik att frontend får filtrere ut de som skal vises? 

Hvis man søker tilbake i tid, er ikke det ett dato som man setter etter att man valg barnen?

Reply:
Jo, så _kan_ hende at man vil ha med alle barna, men de kan jo legge inn barnet selv i verste fall...